test.js

```diff
@@ -0,0 +1,5 @@
+const ext = require('./templatesProvider.js');
+
+const repo = ext.getRepositories();
+
+repo.then(x => console.log(x));
```

Comment:
I suggest we add new line at the end of the file to avoid potential issues if we wan to process/parse the file in the future.